src/test/java/odysseusAT/MyStepdefs.java

```diff
@@ -61,13 +62,13 @@ public static void openProjectPage() throws InterruptedException {
 
     @Then("^check unauthorized user$")
     public void checkUnauthorizedUser() {
-        $(xpath("//*[@id='wrapperMainWindowContainer']/user-bar/div/div/div/div[1]/a")).shouldHave(text("Sign In"));
+        $(xpath("//*[@data-bind='click: function () { $component.signInOpened(true) }']")).shouldHave(text("Sign In"));
```

Comment:
`$(".user-bar-container a").shouldHave(text("Sign In"));`
I think that's use selenide selectors better for read and less depends on code changes